README.md

```diff
@@ -40,131 +40,152 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
+    kubectl apply -f k8s.yml
+    ```
+   
+#### Generated Kubernetes resources
+
+| Kind | Name | Description
+|---|---|---
+| [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) | imagetagger | The namespace in which each resource resides
+| [Deployment](https://kubernetes.io/es/docs/concepts/workloads/controllers/deployment/) + [Service](https://kubernetes.io/docs/concepts/services-networking/service/) | imagetagger-postgres | postgresql server which is required by ImageTagger. The replica count can be dialed down to 0 if an the usage of an external server is desired.
+| [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) | imagetagger-database | Where the postgresql server stores its data
+| [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) | imagetagger-postgres | Configuration of the postgresql server. Also available inside the application server deployment so that settings can be [referenced](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/).
+| [Deployment](https://kubernetes.io/es/docs/concepts/workloads/controllers/deployment/) + [Service](https://kubernetes.io/docs/concepts/services-networking/service/) | imagetagger-web | application server. Per default this deployment references the image `imagetagger:local` which is probably not resolvable and should be replaced by a reference to where your previously built container image is available.
+| [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) | imagetagger-image-data | Where the application server stores its images (and tools).
+| [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) | imagetagger-web | Configuration of the application server. Mounted as environment variables.
+
+## Configuration
+
+ImageTagger is a Django application and uses [django-configurations](https://django-configurations.readthedocs.io/en/stable/)
+for better configuration management.
+
+It is configured to use a *Dev* configuration when running locally and *Prod* when running in a container.
+This can be overridden via the environment variable `DJANGO_CONFIGURATION`.
+
+For a list of available configuration values see [settings.py](https://github.com/bit-bots/imagetagger/blob/master/imagetagger/imagetagger/settings.py). 
+Towards the bottom is a list of *values*. These are taken from environment variables where the key is the variable name
+but with an `IT_` prefix.
+
+If completely custom configuration is desired, `imagetagger/imagetagger/settings_local.py` can be created in which
+a custom configuration class may be created. In docker this file may be located at `/app/config/settings.py` so that
+mounting it should be simple.
+To use this custom configuration class, the environment variables `DJANGO_SETTINGS_MODULE=imagetagger.settings_local`
+and `DJANGO_CONFIGURATION=MyCustomClass` must be set.
```

Comment:
I think this section should go before the Kubernetes stuff that is probably irrelevant for most users, maybe even before the setup section. Also, as mentioned in other comments, I would prefer to have a more detailed guide on which settings to adapt (as we had before) instead of giving them a 250 line configuration file and no information on what to adapt.

Reply:
I could accept putting *Configuration* above *Installing and running ImageTagger* but not putting it just above the kubernetes instructions.

Another alternative would be a table of contents which I have implemented in 696b89fcedc12ce9b3a588363b8c112b9dcd98bf